### PR TITLE
Add new issue: gnqa integration to global search design.

### PR DESCRIPTION
**Pull request by Alexanderlacuna**
This is a design proposal for integrating gnqa to global Search.

### CODE REVIEW

**Comment by Alexanderlacuna** [comment]
@jnduli @BonfaceKilz Please review this, I will get started on some of the features.

File: issues/gnqa_integration_to_global_search_Design.gmi [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+### Backend Design
+The API handles requests to the Fahamu API and manages result caching. Once a request to the Fahamu API is successful, the results are cached using SQLite for future queries. Additionally, a separate API is provided to query cached results.
+
+## Deep Dive
+
+### Caching Implementation
+For caching, we will use SQLite3 since it is already implemented for search history. Based on our study, this approach will require minimal space:
+
+*Statistical Estimation:*  
+We calculated that this caching solution would require approximately 79MB annually for an estimated 20 users, each querying the system 5 times a day.
```
Comment (jnduli) [inline issues/gnqa_integration_to_global_search_Design.gmi:31]: More details would help here. My suggestion would be to add:

1. average request size per user and how we determined this
2. why we're assuming 20 users making 5 requests per day

Comment (Alexanderlacuna) [reply issues/gnqa_integration_to_global_search_Design.gmi:31]: The average request size was an upper bound calculation for documents returned from the Fahamu API. We’re assuming 20 users making 5 requests per day to estimate typical usage of GN2 services.

Comment (Alexanderlacuna) [reply issues/gnqa_integration_to_global_search_Design.gmi:31]: PS:  Apologies,  Got your question wrong, But this is now captured in the docs.

**Review by jnduli — APPROVED** [review]

**Pull request merged**